### ISSUE TITLE
[BUG] Radeon 6800 / 6900 XT locked mem at 192MHz

### ISSUE THREAD

**Issue by Chiqui1234ok** [bzminer/bzminer#100]
Hi! I have a RX 6900 XT and RX 6800 in different rigs, same AMD 22.5.1 driver 😁
In both rigs the memory clock of Radeon cards locks at 192MHz therefore performance is low. The measurement of this is correct because I double checked with GPU-Z 👍

I try to set overclock with Adrenaline Radeon's driver with no success. After that, I installed MSI Afterburner and issue continues.

[Image](https://i.imgur.com/3klBoUW.png)

**Comment by EricKeong** [comment]
Same here rx6800 non xt miner started as cfg 1cl or 28cl on 1 of the 6 card hung gpu detected and no hashrate, the other 5 card no problem with cfg 30cl.

**Comment by pdieppa** [comment]
What is your core clock and core voltage for the 6800?

**Comment by Chiqui1234ok** [comment]
> What is your core clock and core voltage for the 6800?

Both my 6900 XT and 6800 I have 1500MHz Core, 2050/2100MHz Mem. Same bug, no matter if I changed the OC while mining

**Comment by EricKeong** [comment]
As my core clock 1800 vdd(core voltage) 700 for kaspa, it keep gpu hung and restart no hashrate. Now i have get my working already refer to alph overclock, just adding vdd(core voltage) to 725. So i think its the core voltage not enuff for core clock 1800 in my cases.

**Comment by pdieppa** [comment]
> As my core clock 1800 vdd(core voltage) 700 for kaspa, it keep gpu hung and restart no hashrate. Now i have get my working already refer to alph overclock, just adding vdd(core voltage) to 725. So i think its the core voltage not enuff for core clock 1800 in my cases.

You are correct. It needs higher VDD is order to support the high clock. I run my 6800 XT at 1550 core and 645 VDD hashing at 537mh with a software reported watts of 84.  I'm going for efficiency since my outlet does not support running all the cards at their max.

**Comment by Chiqui1234ok** [comment]
Even at stock clock (without any OC), my mem locks at 192MHz...

**Comment by Chiqui1234ok** [comment]
Anyone? :(

**Comment by bzminer** [comment]
hey @Chiqui1234ok sorry for the delayed reply. bzminer doesn't actually overclock amd right now, so that sounds like a software outside of bzminer setting your amd clocks. do you have them set in a mining os?
